### PR TITLE
16382 - add name to limited restoration certificate

### PR DESCRIPTION
**Pull request by ketaki-deodhar**
*Issue #:* /bcgov/entity#16382

*Description of changes:*

Some examples:
Before:
image

After fix:
image

Before:
image

After fix:
image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

**Review by ArwenQin — APPROVED** [review]
Looks good

File: legal-api/report-templates/certificateOfRestoration.html [hunk 1]
```diff
@@ -35,7 +35,8 @@
                 <span class="bold-italic">{{business.legalName}}</span>
               {% elif restoration.type == 'limitedRestoration' %}
                 has on {{effective_date_time}} been restored for a period ending
-                <span class="bold-italic">{{restoration_expiry_date}}</span> under the {{entityAct}}.
+                <span class="bold-italic">{{restoration_expiry_date}}</span> under the {{entityAct}} as
+                <span class="bold-italic">{{business.legalName}}</span>
```
Comment (vysakh-menon-aot) [inline legal-api/report-templates/certificateOfRestoration.html:39]: @severinbeauvais This is not a bug. Just realized it was the requirement. Does the requirement change? 
Ref: https://preview.uxpin.com/306da47387722817e24fc77fd71476a1869a05fe#/pages/160343881/simulate/no-panels

Comment (severinbeauvais) [reply legal-api/report-templates/certificateOfRestoration.html:39]: Good catch,

I've asked Mihai about this specific scenario, which isn't shown in the UI designs. (Ie, a limited restoration where the historical business name is different from the limited-active business name.)

Comment (vysakh-menon-aot) [reply legal-api/report-templates/certificateOfRestoration.html:39]: Do we need the same logic in limitedRestorationExtension and limitedRestorationToFull?

**Comment by ketaki-deodhar** [comment]
@vysakh-menon-aot @severinbeauvais okay to go ahead with this change?

File: legal-api/report-templates/certificateOfRestoration.html (hunk 1 above)
Comment (severinbeauvais) [reply legal-api/report-templates/certificateOfRestoration.html:39]: Probably.

Comment (ketaki-deodhar) [reply legal-api/report-templates/certificateOfRestoration.html:39]: I will check with Mihai

Comment (severinbeauvais) [reply legal-api/report-templates/certificateOfRestoration.html:39]: @ketaki-deodhar , check with Mihai about the other 2 resto scenarios. If there are no more changes then go ahead and merge.

**Comment by ketaki-deodhar** [comment]
@vysakh-menon-aot @severinbeauvais created new ticket [23405](https://app.zenhub.com/workspaces/entities---olga-65af15f59e89f5043c2911f7/issues/gh/bcgov/entity/23405)
One of you will have to approve this PR to be merged.

**Review by vysakh-menon-aot — APPROVED** [review]

**Pull request merged**